doc/dash/dash-sonic-hld.md

```diff
@@ -573,26 +579,26 @@ DASH_VNET_MAPPING_TABLE:{{vnet}}:{{ip_address}}
     "routing_type": {{routing_type}} 
     "underlay_ip":{{ip_address}}
     "mac_address":{{mac_address}} (OPTIONAL) 
-    "metering_class": {{class_id}} (OPTIONAL)
-    "override_meter": {{bool}} (OPTIONAL)
+    "metering_class_or": {{uint32}} (OPTIONAL)
     "use_dst_vni": {{bool}} (OPTIONAL)
     "use_pl_sip_eni": {{bool}} (OPTIONAL)
-    "overlay_sip":{{ip_address}} (OPTIONAL)
-    "overlay_dip":{{ip_address}} (OPTIONAL)
-    "routing_appliance_id": {{uint32}} (OPTIONAL)
+    "overlay_sip_prefix":{{ip_prefix}} (OPTIONAL)
+    "overlay_dip_prefix":{{ip_prefix}} (OPTIONAL)
+    "routing_appliance_id": {{uint32}} (OPTIONAL) (OBSOLETED)
+    "tunnel": {{string}} (OPTIONAL)
 ```
 ```
 key                      = DASH_VNET_MAPPING_TABLE:vnet:ip_address ; CA-PA mapping table for Vnet
 ; field                  = value 
 action_type              = routing_type              ; reference to routing type
 underlay_ip              = ip_address                ; PA address for the CA
 mac_address              = MAC address as string     ; Inner dst mac
-metering_class           = class_id                  ; metering class-id
-override_meter           = bool                      ; override the metering class-id coming from the route table
+metering_class_or        = uint32                    ; metering class 'or' bits
```

Comment:
Does this table need a `metering_class_and` field?

Reply:
`metering_class_and` is only required for route tables currently.